spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaReactiveWebServerFactory.java

```diff
@@ -192,9 +193,13 @@ public WebServer getWebServer(HttpHandler httpHandler) {
                                                   Ints.saturatedCast(minResponseSize)));
         }
 
+        final DocServiceBuilder docServiceBuilder = DocService.builder();
+        findBeans(DocServiceConfigurator.class).forEach(
+                configurator -> configurator.configure(docServiceBuilder));
```

Comment:
If fails to find `DocServiceConfigurator`, how about calling `configureArmeriaService(sb, settings)`?

Reply:
Is it possible to add a test case for `DocServiceConfigurator` to prevent future mistakes?

Reply:
> If fails to find `DocServiceConfigurator`, how about calling `configureArmeriaService(sb, settings)`?

Could you explain it in more detail?
I referred to [ArmeriaAutoConfiguration](https://github.com/line/armeria/blob/master/spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaAutoConfiguration.java#L108-L111).

Reply:
My explanation was not enough. 😅 What I meant was:
```java
List<DocServiceConfigurator> docServiceConfigurators = findBeans(DocServiceConfigurator.class);
if (docServiceConfigurators.isEmpty()) {
   configureArmeriaService(sb, settings);
} else {
   final DocServiceBuilder docServiceBuilder = DocService.builder();
   docServiceConfigurators.forEach(
       ... // Same as you did 
   );
}
```

Reply:
Thank you for your explanation.
I need to pass DocServiceBuilder to configureArmeriaService if I can find DocServiceConfigurator.
You mean that would it be better to create overload method like the following?


```java
final ArmeriaSettings settings = findBean(ArmeriaSettings.class);
if (settings != null) {
    final List<DocServiceConfigurator> docServiceConfigurators =
            findBeans(DocServiceConfigurator.class);
    if (docServiceConfigurators.isEmpty()) {
        configureArmeriaService(sb, settings);
    } else {
        final DocServiceBuilder docServiceBuilder = DocService.builder();
        configureArmeriaService(sb, docServiceBuilder, settings);
        docServiceConfigurators.forEach(configurator -> configurator.configure(docServiceBuilder));
    }
}

private void configureArmeriaService(ServerBuilder sb, ArmeriaSettings settings) {
    final DocServiceBuilder docServiceBuilder = DocService.builder();
    configureArmeriaService(sb, docServiceBuilder, settings);
}

private void configureArmeriaService(ServerBuilder sb, DocServiceBuilder docServiceBuilder,
                                        ArmeriaSettings settings) {
    ...
}
```

Reply:
The current code looks OK to me.  `DocService` has to be built even if there are no `DocServiceConfigurator`, so there's no point of handling the case of empty list specially.

Reply:
Let me mark this thread as resolved. No change is necessary.

Reply:
Sorry for the wrong direction. 😭 I thought `configureArmeriaService` is an API that is outside of this code.
Please ignore this review. Your code was good from the start! 🙇‍♂️